go/oasis-node/main.go

```diff
@@ -2,9 +2,14 @@
 package main
 
 import (
+	"github.com/hashicorp/go-plugin"
+
 	"github.com/oasisprotocol/oasis-core/go/oasis-node/cmd"
 )
 
 func main() {
+	// If we use go-plugin, we are supposed to clean clients up.
```

Comment:
Probably the same in remote signer?

Reply:
And in the tests yeah.  I'll deal with that after I unbreak the tests.